config/cloud_controller.yml

```diff
@@ -393,3 +393,4 @@ max_manifest_service_binding_poll_duration_in_seconds: 60
 update_metric_tags_on_rename: true
 
 app_log_revision: true
+app_stopping_state: true
```

Comment:
**thought** Should this be process_stopping_state? Since the app `state` field is the desired state.